server/feeders/reviewsgenerator.js

```diff
@@ -107,16 +109,24 @@ var triggerRestaurantsRatings = function() {
   var servicePath;
 
   var q = async.queue(function(task, callback) {
-
+    var restaurantId = utils.asciiEncode(
+      utils.stripForbiddenChars(
+        unescape(task.restaurantName)));
```

Comment:
I've gone crazy with this 'encoding' thing. I've tested everything, and that was the only way I could perform all the requests (for the 688 restaurants). Here are my tries:

- Using the following code:
```js
var restaurantId = utils.stripForbiddenChars(task.restaurantName);
```
It queries:
```
http://orion:1026/v2/entities/Se%C3%B1orio_de_Astobiza_%2528Okendo_Txakolina%2529
``` 
Which returns:
```js
{ error: 'BadRequest', description: 'invalid character in URI' }
```
Here's the first issue: the `ñ` (note that this behaviour is the same for `á, é, í` and so on). As stands in the specification, field sintax restrictions, it says 'Allowed characters are the ones in the plain ASCII set'. Then I made the function asciiEncode and try again:

```js
var restaurantId = utils.asciiEncode(utils.stripForbiddenChars(task.restaurantName));
```
This queries:
```
http://orion:1026/v2/entities/Se%5Cu00f1orio_de_Astobiza_%2528Okendo_Txakolina%2529
```
And returns the following:
```js
{ error: 'NotFound',
  description: 'The requested entity has not been found. Check type and id' }
```
This is because, the entity id is loaded in Orion as `Señorio_de_Astobiza__Okendo_Txakolina_`, but the name, the variable `task.restaurantName` is the name attribute of that id stored in Orion. The thing is, to store the original restaurant name before, we had to URI encode it (name), so the name is stored in Orion entity as `Señorio de Astobiza %28Okendo Txakolina%29`, otherwise it returns the following error:
```js
{ error: 'BadRequest',
        description: 'Invalid characters in attribute value' } } }
```
Due to that, the only thing we could do to accomplish this is:
- Get the name from the Restaurant entity (URI encoded)
- Unescape it (or `decodeURIComponent` should work also but I did not test it)
- Strip the forbidden characters
- Ascii encode the rest of them

With that procedure we can conclude in:
```
var restaurantId = utils.asciiEncode(utils.stripForbiddenChars(unescape(task.restaurantName)));
```
Which does the following request:
```
http://orion:1026/v2/entities/Se%5Cu00f1orio_de_Astobiza__Okendo_Txakolina_
```
And finally returns the entity we want to.

I do know this is a mess and I'm the first one I dislike this procedure, but it was the only way I found to load all the restaurants.